vcbuild.bat

```diff
@@ -497,11 +497,11 @@ for /d %%F in (test\addons-napi\??_*) do (
   rd /s /q %%F
 )
 :: building addons-napi
-for /d %%F in (test\addons-napi\*) do (
-  %node_gyp_exe% rebuild ^
-    --directory="%%F" ^
-    --nodedir="%cd%"
-)
+setlocal
+set npm_config_nodedir=%~dp0
+"%node_exe%" "%~dp0tools\build-addons.js" "%~dp0deps\npm\node_modules\node-gyp\bin\node-gyp.js" "%~dp0test\addons-napi"
+if errorlevel 1 exit /b 1
+endlocal
 endlocal
```

Comment:
Could you remove this extra `endlocal`.
Any idea what this mathes? My guess it's probably left over from something old https://github.com/nodejs/node/commit/c403eeb7fdf8c1033422b6ea8bf025667892f867

Reply:
TBH, I don't know. It's there, I don't want to break anything. Another PR maybe?

Reply:
NP.

Reply:
Ping me when you land this.